.github/workflows/ci.yml

```diff
@@ -86,7 +86,7 @@ jobs:
         with:
           platforms: arm64,ppc64le
       - name: Test Docker Image
-        run: core/docker/build.sh
+        run: core/docker/build.sh -c
```

Comment:
> Some issues with stability of Ubuntu repositories have surfaced lately.
Our Dockerfile tries to update packages in the base Ubuntu image, and
when that fails, the entire `maven-checks` job fails.


Updates are not necessary (per Docker's philosophy, updates to the base image should be part of base image lifecycle), but we  install new stuff too.

> We can use layers from the latest trino image to avoid running
these updates.

We already have some retries
https://github.com/trinodb/trino/blob/257c2a5ac47a731651b43cc7af35f83bf5a034c4/core/docker/Dockerfile#L29-L30
are these not sufficient?
what can we do to make them sufficient?

This change may get CI working, but we cannot rely on that for automated releases (once we have them), so would be nice to have some more self-contained solution

Reply:
BTW this means CI won't see any updates to `eclipse-temurin:17-jdk` image only until we do a release.
So if `eclipse-temurin:17-jdk` updates affects us in any way, we won't know about that until our users know about that.